wger/nutrition/api/views.py

```diff
@@ -167,11 +169,22 @@ def search(request):
         )
 
         for ingredient in ingredients:
+            if hasattr(ingredient, 'image'):
```

Comment:
can't we just do `if ingredient.image:` here?

Reply:
I tried this but in that case RelatedObjectDoesNotExist is raised when the image is not there. The recommended way by Django to work around this is by using hasattr (according to https://docs.djangoproject.com/en/4.0/topics/db/examples/one_to_one/).

Reply:
you are right